public/Day 61/src/client/app/(private)/(chat)/ChatInput.tsx

```
@@ -0,0 +1,195 @@
+      const recorder = new MediaRecorder(stream);
+      setMediaRecorder(recorder);
+      audioChunks.current = [];
+
+      recorder.ondataavailable = (e) => {
+        audioChunks.current.push(e.data);
+      };
+
+      recorder.onstop = () => {
+        const audioBlob = new Blob(audioChunks.current, { type: "audio/mp3" });
+        setAudioBlob(audioBlob);
+        stream.getTracks().forEach((track) => track.stop());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)